secure_message/common/utilities.py

```diff
@@ -113,11 +135,15 @@ def add_from_details(messages):
     for user in party.get_users_details(get_external_user_uuid_list(messages)):
         external_user_details[user['id']] = user
     for message in messages:
-        if message["from_internal"]:
-            message.update({"@msg_from": internal_user_service.get_user_details(message["msg_from"])})
-        else:
-            if external_user_details.get(message['msg_from']):
-                message.update({'@msg_from': external_user_details.get(message['msg_from'])})
+        try:
+            if message["from_internal"]:
+                message.update({"@msg_from": internal_user_service.get_user_details(message["msg_from"])})
+            else:
+                if external_user_details.get(message['msg_from']):
+                    message.update({'@msg_from': external_user_details.get(message['msg_from'])})
+        except IndexError:
+            logger.exception(f"Exception adding from details message:'{json.dumps(message)}'")
```

Comment:
These should be key values pairs to be consistent with the software engineering community standards: https://github.com/ONSdigital/software-engineer-community/blob/master/standards/logging.md

Reply:
oo. Good catch I took them out of one func and missed the other changing